src/event-listeners/index.js

```diff
@@ -29,11 +31,15 @@ import {
   closeGoalForm,
   updateContent,
 } from '../goal-form/actions'
-import { archiveGoal } from '../goals/actions'
+import { archiveGoal, createGoal } from '../goals/actions'
 import { setScreenDimensions } from '../screensize/actions'
 import { changeTranslate, changeScale } from '../viewport/actions'
 
 import layoutFormula from '../drawing/layoutFormula'
+import { setGoalClone } from '../goal_clone/actions'
+import moment from 'moment'
+import { addMemberOfGoal } from '../goal-members/actions'
+import cloneGoals from '../components/cloneGoals'
```

Comment:
Can you move `cloneGoals.js` file into the same folder as this file I guess. `event-listeners`